ts/navigation/TabNavigator.tsx

```diff
@@ -162,20 +181,23 @@ export const MainTabNavigator = () => {
             )
           }}
         />
+        {isNewProfileVisibile && (
```

Comment:
It could be a good practice when a new feature is introduced to start defining a local feature flag? Like this feature https://github.com/pagopa/io-app/blob/011f5689e6a9a88620560e61e605858abd842f2b/ts/config.ts#L98 a local ff was defined to anbled/disabled locally a feature.